scratchapi2/excs.py

```diff
@@ -1,9 +1,5 @@
-""" Scratch API Error """
+"""Scratch API errors"""

```

Comment:
No need for title case in docstrings. Leading and trailing spaces are also an abomination.